content/c-sharp/concepts/operators/operators.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: 'Operators'
+Description: 'Operators are used to perform various operations on variables and values.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Operators'
+  - 'Arithmetic'
+  - 'Comparison'
+  - 'Logical'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+**Operators** are used to perform various operations on variables and values.
+
+## Syntax
+
+The following code snippet uses the assignment operator, `=`, to set `my_variable` to the value of `num1` and `num2` with an arithmetic operator acting on them. For example, if `operator` represented `*`, `my_variable` would be assigned a value of `num1 * num2`.
+
+```pseudo
+my_variable = num1 operator num2;
+```
+
+Operators can be organized into the following groups:
+
+- Arithmetic operators for performing traditional math evaluations.
+- Assignment operators for assigning values to variables.
+- Comparison operators for comparing two values.
+- Logical operators for combining Boolean values.
+- [Bitwise operators](https://www.codecademy.com/resources/docs/general/bitwise-operator) for manipulating the bits of a number.
+
+## Arithmetic Operators
+
+C# has the following arithmetic operators:
+
+- Addition, `+`, which returns the sum of two numbers.
+- Subtraction, `-`, returns the difference between two numbers.
+- Multiplication, `*`, returns the product of two numbers.
+- Division, `/`, returns the quotient of two numbers.
+- Modulus, `%`, returns the remainder of one number divided by another.
+
+The ones above operate on two values. C# also has two unary operators:
+
+- Increment, `++`, which increments its single operand by one.
+- Decrement, `--`, which decrements its single operand by one.
+
+Unlike the other arithmetic operators, the increment and decrement operators change the value of their operand as well as return a value. They also return different results depending on if they precede or follow the operand. Preceding the operand returns the value of the operand before the operation. Following the operand returns the value of the operand after the operation.
+
+```cs
+int x = 5;
+int y = x++; // x now equals 6, y equals 5
+int z = --x; // x now equals 5, z equals 5
+```
+
+## Logical Operators
```

Comment:
There should be a mention here that logical operators ALWAYS evaluate both operands and do not "short-circuit" them like a conditional operator.